client/src/App.js

```diff
@@ -13,73 +14,35 @@ function Berries() { return <h2> Berries page </h2>; }
 function NoMatch() { return <h2> Page does not exist </h2>; } 
 
 class App extends Component { 
-  
-  state = {
-    data: null,
-    links: [
-      { url: "/", text: "Home", idStr: "home-link" },
-      { url: "/Pokemon", text: "Pokemon", idStr: "pokemon-link" },
-      { url: "/Moves", text: "Moves", idStr: "moves-link" },
-      { url: "/Items", text: "Items", idStr: "items-link"},
-      { url: "/Berries", text: "Berries", idStr: "berries-link" }
-    ],
-
-    activeLink: "home-link"
-  }; 
-
-  /* Event handlers */
-  handleLinkClick(e){
-    e.preventDefault();
-    var idStr = e.currentTarget.id;
-    this.setState({activeLink: idStr});
-  } 
-
-  /* Helper functions */
-  renderLinks(){
-    var linkArr = this.state.links.slice();
-
-    var links = linkArr.map((curr) => {
-      var idStr = curr.idStr;
-      var classStr = ( (idStr === this.state.activeLink) ? "active-link": ""); 
-
-      return (
-        <li className={classStr}
-            id={curr.idStr}
-            key={curr.idStr}
-            onClick={(e) => this.handleLinkClick(e)}>
-
-          <Link to={curr.url}> {curr.text} </Link>
```

Comment:
Just as an FYI, `<Link>`s are already `<a>` elements. So when you were clicking on this button, you were clicking on Link (`<a>`'s href) not on `<li>`'s `onClick`. So old lines 46-49 were unnecessary.